### DISCUSSION TITLE
Add collapse button to track groups in the Animation editor

### DISCUSSION THREAD

**Discussion by joancardonasa** [godotengine/godot-proposals#10555] [Editor]
_Edit: I just realized another [discussion ](https://github.com/godotengine/godot-proposals/discussions/8655)is open regarding this, but I feel my proposal offers a bit more information and can prove more useful._

**Describe the project you are working on:**
A 3D isometric action-adventure game.

**Describe the problem or limitation you are having in your project:**
The game I am building relies heavily on animation-driven combat. As such, I spend a lot of time animating characters in Blender, importing them into Godot and tweaking parameters in the Animation editor. Many of their behaviours are driven through methods called via animations with custom tracks.

When adding new custom tracks to the animations I made, it's hard to navigate the editor because of all the bone information from the Skeleton3D node, which is mostly irrelevant when I'm adding new custom tracks. I know there's an option to only visualize the Node that's currently selected, but this becomes cumbersome when there are several components you're juggling between.

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
The idea would be to add a small collapse button next to the Node containing the tracks to help navigate the Animation editor more easily.

![image](https://github.com/user-attachments/assets/8852703a-f469-4082-921c-c5065d484033)

**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
No, as it's an editor issue.

**Is there a reason why this should be core and not an add-on in the asset library?:**
As Godot gets more attention from people making 3D games, this UX improvement can make their lives easier and help Godot's animation tools be at the same level as Blender's or other general purpose game engines.

**Comment by AThousandShips** [comment]
Thank you for your proposal, consolidating in:
* https://github.com/godotengine/godot-proposals/issues/10276
